experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -64,6 +63,22 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
     return sim
 end
 
+# Extension of CA.set_surface_albedo! to set the surface albedo to 0.38 at the beginning of the simulation,
+# so the initial callback initialization doesn't lead to NaNs in the radiation model.
+function CA.set_surface_albedo!(Y, p, t, ::CA.CouplerAlbedo)
```

Comment:
Do we need to extend this for `ConstantAlbedo` and `RegressionFunctionAlbedo` too, or is that done in Atmos?

Reply:
`water_albedo_from_atmosphere!` below is basically doing the `RegressionFunctionAlbedo` but at a different time than in atmos. I can clarify this in the comment :)